karabo_bridge/cli/simulation.py

```diff
@@ -37,14 +38,18 @@ def main(argv=None):
         '-g', '--gen', default='random', choices=['random', 'zeros'],
         help='Generator function to generate simulated detector data'
     )
-
+    ap.add_argument(
+        '--data_like', default='online', choices=['online', 'file'],
```

Comment:
As far as I know, command-line options always use `-` rather than `_`.

(OK, there's bound to be some exceptions, but that's definitely the common pattern.

Reply:
Right, thanks